internal/archive/archive.go

```diff
@@ -39,6 +39,8 @@ type Options struct {
 	Pro        string
 	CacheDir   string
 	PubKeys    []*packet.PublicKey
+	// This Ubuntu archive is no longer actively maintained.
+	Unsupported bool
```

Comment:
The phrasing here doesn't seem ideal. Besides it being a boolean, which means we cannot express a gradient if we have to, it's also a negative boolean, which means we need to test for "not unsupported" when we mean supported.  Need to sync with Chris on this one.